scripts/jenkins/ardana/ansible/vars/provider.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+##
+## For running openstack command
+##
+openstack_cmd_args_g: >
+  --os-username {{ glance_admin_user }}
+  --os-project-name {{ admin_project_name }}
+  --os-user-domain-name Default
+  --os-project-domain-name Default
+  --os-auth-url {{ glance_auth_url }}
+  --os-interface internal
+  --os-cacert {{ ca_file }}
+
+openstack_cmd_args_h: >
```

Comment:
same question here.

nit1: This is very similar to the first call, can we just use one of them, and adapt if need be?
nit2: As both are only used once in the play, can those 2 vars be removed, and directly set in the play? It would be so easier to track and read.